### ISSUE TITLE
Poster image is not shown on iphone device.

### ISSUE THREAD

**Issue by mahmutbayri** [hola/hola_player#15]
Hi,
I tested the problem at http://hola.github.io/examples/cdn/#hola_player_cdn. when i try to use like at below, poster image is not shown on iphone device. 

```
    (function(){
        window.hola_player({
        	poster: "poster image url"
        });
    })();
```

it appears when div container element of display status  is set to block !important. 

Thanks.

**Comment by norlin** [comment]
Hi,

The issue is fixed now (v1.0.62)

Thank you!

**Comment by mahmutbayri** [comment]
Thanks.
